pwa/src/components/subscribers/subscriberForm.tsx

```diff
@@ -42,14 +38,22 @@ export const SubscriberForm: React.FC<SubscriberFormProps> = ({ subscriberId, en
   const [sources, setSources] = React.useState<any>(null);
   const [tableNames, setTableNames] = React.useState<Array<any>>(null);
```

Comment:
In this declaration, we're doing two things that are not how we do it in any other of our declarations:

1. We're using the `Array` keyword instead of the `[]` syntax and
2. We're specifying that `tableNames` is an array at all.

Other declarations we do like:

`const [sources, setSources] = React.useState<any>(null);`

And if we were to add that this is an array, we would usually do it like:

`const [sources, setSources] = React.useState<any[]>(null);`

For now, I'd just make the declaration like this:

`const [tableNames, setTableNames] = React.useState<any>(null);`